apps/site/components/Containers/NavBar/index.module.css

```
@@ -28,12 +28,18 @@
     dark:border-neutral-900;
 }
 
+.sidebarItemToggler {
+  @apply absolute right-4 -z-10 appearance-none opacity-0;
+  transform: translateY(-200%);
```

```suggestion
  transform: translateY(-200%);
```

https://tailwindcss.com/docs/translate

---

@canerakdas how would it be in this scenario?

---

I think it is enough to use `-translate-y-[200%]` as there is no special value for -200%